client/src/components/Reviews/components/ReviewList.test.js

```diff
@@ -0,0 +1,23 @@
+import { render, fireEvent, waitFor, screen } from '@testing-library/react';
```

Comment:
Don't need to import render. Our test setup script does that for us. Also don't need to import fireEvent since we're using userEvent anyways.

Reply:
Also looks like `waitFor` isn't needed either. If there's any imports that you do find are necessary, feel free to add them to `testSetup.js`. It'll eliminate the need for repetitive boilerplate in our test files.

Reply:
Yeah, I had copied this over from the docs when I was starting and never changed it

Reply:
Oh ok makes sense.